<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SIFT Demo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="browse_button">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>470</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse template...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="toggle_cam_button">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>470</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Enable camera</string>
    </property>
   </widget>
   <widget class="QLabel" name="template_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>320</width>
      <height>240</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="live_image_label">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>130</y>
      <width>320</width>
      <height>240</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
